MicrosoftFluentUI.podspec

```diff
@@ -183,6 +183,14 @@ fi', :execution_position => :before_compile }
     label_ios.source_files = ["ios/FluentUI/Label/**/*.{swift,h}"]
   end
 
+  s.subspec 'ListItem_ios' do |listitem_ios|
+    listitem_ios.platform = :ios
```

Comment:
Should this include visionOS?

Reply:
Hmm I'm not sure actually, we don't specify visionos in any of the other subspecs, I assume some of the other pods are being used in vision and they didn't need to specify visionos. @joannaquu @mischreiber do either of you know whether we need to specify it?

Reply:
Will leave as is for now, and follow up if needed

Reply:
Sounds like we may not be allowing cocoapods users of FUA to use visionOS at all, which is outside the scope of this PR. Fair for followup :)

Reply:
That's correct, we haven't dealt with visionOS and Cocoapods yet.